src/db/migrations/20170929203228_add_prefix_url/down.sql

```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Can you make this remove the prefix_url column from the table? I would like to keep all up->down combinations no-op. Removing a column in SQLite is not trivial but there is an example of such a migration already, in directories_date_added.